### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bishthimesh**

### CODE REVIEW

**Comment by XhmikosR** [comment]
@mdo Hackoctoberfest stuff, so if it's invalid just add the label and close it.

**Comment by bishthimesh** [comment]
> @mdo Hackoctoberfest stuff, so if it's invalid just add the label and close it.

Its not invalid. I have corrected typo and grammar.

**Comment by XhmikosR** [comment]
Either way, a native English speaker will have a look later.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 ## Bootstrap 4
 
-Our default branch is for development of our upcoming Bootstrap 5 release. Head to the [`v4-dev` branch](https://github.com/twbs/bootstrap/tree/v4-dev) to view the readme, documentation, and source code for Bootstrap 4.
```
Comment (therealklanni) [inline README.md:27]: Both statements (original and new) are grammatically correct. This change is superfluous.

**Comment by mdo** [comment]
Yeah, not necessary.

**Pull request closed without merging**